namespace/id.go

```diff
@@ -28,3 +31,8 @@ func (nid ID) Size() IDSize {
 func (nid ID) String() string {
 	return string(nid)
 }
+
+// HexString returns hexadecimal encoding of nid.
+func (nid ID) HexString() string {
+	return hex.EncodeToString(nid)
```

Comment:
[blocking] I think Codecov may block this PR until a unit test is added to cover this line

<img width="369" alt="Screenshot 2023-05-25 at 10 29 41 AM" src="https://github.com/celestiaorg/nmt/assets/3699047/856d6386-c231-454c-8c17-0bb1d1e553e1">

Otherwise LGTM